sng_freepbx_debian_install.sh

```diff
@@ -159,159 +162,206 @@ install_asterisk() {
 		"voicemail"
 	)
 
-	# creating directories 
+	# creating directories
 	mkdir -p /var/lib/asterisk/moh
-	pkg_install asterisk$astver
 
+	asteriskPkgs=""
 	for i in "${!ASTPKGS[@]}"; do
-		pkg_install asterisk$astver-${ASTPKGS[$i]}
+		asteriskPkgs="${asteriskPkgs} asterisk$astver-${ASTPKGS[$i]}"
 	done
 
-	pkg_install asterisk$astver.0-freepbx-asterisk-modules
-	pkg_install asterisk-version-switch
-	pkg_install asterisk-sounds-*
+    allAsteriskPkg=("asterisk$astver ${asteriskPkgs} asterisk$astver.0-freepbx-asterisk-modules asterisk-version-switch asterisk-sounds-*")
+
+	pkg_install ${allAsteriskPkg[*]}
```

Comment:
if we are reverting pkg_install  then this also needs to review again.

Reply:
As per my previous comment I do believe it is better to keep it like that and even remove the force overwrite and foce newconf.